util/tracing/visualization/fedsd.py

```diff
@@ -329,3 +441,5 @@ def load_and_process_csv_file(csv_file) :
 
     # Write to a csv file, just to double check
     trace_df.to_csv('all.csv', index=True)
+    print('Fedsd: Successfully generated the sequence diagram in trace_svg.html.')
+    
```

Comment:
```suggestion

```

Reply:
A number of messages are there to see the progress.
Why remove this final one? An output example looks like this:
```
Fedsd: Successfully converted trace file rti.lft to rti.csv.
Fedsd: Successfully converted trace file federate__mid.lft to federate__mid.csv.
Fedsd: Successfully converted trace file federate__up.lft to federate__up.csv.
Fedsd: Successfully converted trace file federate__down.lft to federate__down.csv.
Fedsd: Successfully generated the sequence diagram in trace_svg.html.
```

Reply:
I'm not suggesting to remove the message, I'm suggesting to remove the excess white space below it. Did I click on the wrong line?

Reply:
The convention is for the last char to be a newline, so no extra spaces.

Reply:
GitHub marks deviations from this "end with newline" practice with red error markers in the diff. I recommend scanning the diff in the PR for obvious things like these before marking the PR ready for review.